rust/bin-shared/src/tun_device_manager.rs

```diff
@@ -12,121 +12,3 @@ pub use windows as platform;
 
 #[cfg(any(target_os = "linux", target_os = "windows"))]
 pub use platform::TunDeviceManager;
-
-#[cfg(test)]
-#[cfg(any(target_os = "linux", target_os = "windows"))]
-mod tests {
-    use super::*;
-    use ip_network::Ipv4Network;
-    use socket_factory::DatagramOut;
-    use std::{
-        net::{Ipv4Addr, Ipv6Addr, SocketAddr, SocketAddrV4},
-        time::Duration,
-    };
-    use tokio::io::{AsyncReadExt as _, AsyncWriteExt as _};
-
-    #[tokio::test]
-    #[ignore = "Needs admin / sudo and Internet"]
-    async fn tunnel() {
-        let _guard = firezone_logging::test("debug");
-
-        no_packet_loops_tcp().await;
-        no_packet_loops_udp().await;
-        tunnel_drop();
-    }
```

Comment:
I moved these tests into a separate test binaries. Those are executed serially by cargo as individual processes so we don't have to sequence them here manually.